eclass/qt5-build.eclass

```diff
@@ -631,7 +631,7 @@ qt5_base_configure() {
 		-no-compile-examples
 
 		# disable rpath on non-prefix (bugs 380415 and 417169)
-		$(usex prefix '' -no-rpath)
+		$(!in_iuse prefix || !use_if_iuse prefix && echo -no-rpath)
```

Comment:
honestly, this is unreadable :P
Doesn't `use_if_iuse` include the `in_iuse` logic?
Also, `use_if_iuse` is defined in eutils.eclass right? we don't inherit that eclass...
I would just stick to `in_iuse`+`usex`.

Reply:
can't comfortably do `usex` if `-no-rpath` shall be the default for `!in_iuse` as well as `!use_if_iuse` cases. but here's a new iteration.